test/functionalTest/cases/2015_notification_templates/notification_templates.test

```diff
@@ -337,7 +337,7 @@ User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
 Ngsiv2-Attrsformat: normalized
 Host: 127.0.0.1:REGEX(\d+)
 Accept: application/json
-Content-Type: application/json; charset=utf-8
+Content-Type: text/plain
```

Comment:
Note this notification is `Ngsiv2-Attrsformat: normalized`. Thus `Content-Type: application/json; charset=utf-8` should be used.

In addition, not sure if we should use "text/plain" or "text/plain; charset=utf-8" for the "custom" default... I'd say that the charset used at text level is the same in both cases, so if we are using charset=utf-8 in JSON case, we should do the same in the text plain case.